packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/data_sources/remote/http_clients/api_http_client.dart

```diff
@@ -10,7 +11,25 @@ class ApiHttpClient with DioMixin implements Dio {
   ApiHttpClient() {
     options = BaseOptions();
     httpClientAdapter = IOHttpClientAdapter();
+    {{#enable_dev_menu}}
+    if (EnvironmentConfig.enableDevMenu == true) {
```

Comment:
Do we really need two flags for the same thing and can we avoid one of them. For instance this one `EnvironmentConfig.enableDevMenu` ?

Current flags
- EnvironmentConfig.enableDevMenu
- {{#enable_dev_menu}}